### ISSUE TITLE
descjop-uberapp fails with `Cannot run program "cmd.exe"` on MacOS

### ISSUE THREAD

**Issue by DrTom** [karad/lein_template_descjop#29]
We got through all the steps untill it comes to packaging: 

```
lein descjop-uberapp-osx
java.io.IOException: Cannot run program "cmd.exe" (in directory ....
```

This is invoked on MacOS 10.12.

**Comment by DrTom** [comment]
I just realized those are just lein aliases, which reads like the following in project.clj
```
            "descjop-uberapp-osx" ["shell" "cmd.exe" "/c" "electron-packager" "./app/prod" "madek-app" "--platform=darwin" "--arch=x64" "--electron-version=1.6.0"]

```

**Comment by DrTom** [comment]
I just realized these are just aliases in project.clj
```
            "descjop-uberapp-osx" ["shell" "cmd.exe" "/c" "electron-packager" "./app/prod" "madek-app" "--platform=darwin" "--arch=x64" "--electron-version=1.6.0"]
```

so removing ["cmd.exe" "/c"] solved the problem. Should they not be absent by default?

**Comment by karad** [comment]
Thank you for your report! I'll check it!!

**Comment by karad** [comment]
fixed this bug v0.7.2
